### ISSUE TITLE
How to retrieve the grid coords

### ISSUE THREAD

**Issue by Mark-Creeten** [ducksboard/gridster.js#23]
I'm trying to retrieve the coords for the block that just changed place in the grid. how do i do that?

that is what i have so far:

``` javascript
var gridster = $(".gridster ul").gridster({
    widget_margins: [10, 10],
    widget_base_dimensions: [140, 140],
    max_size_y: 3,
    serialize_params: function($w, wgd) {
      return {
        col: wgd.col,
        row: wgd.row
      };
    },
    draggable: {
      stop: function(event, ui) {
        console.info('yep');
      }
    }
```

**Comment by vieron** [comment]
I guess you got it, but would be:

```
stop: function(event, ui) {
    console.log(this.$player.coords().grid);
},
start: function(event, ui) {
    console.log(this.$player.coords().grid);
},
drag: function(event, ui) {
    console.log(this.$player.coords().grid);
}
```

**Comment by Mark-Creeten** [comment]
thanks! for the answer, but that dont work on the stop event. $player is undefined, coord() is not a function

**Comment by vieron** [comment]
@Mark-Creeten it shoud work in the last version of the code. It was fixed in the commit referenced above (573ed66).
